Remotion/ObjectBinding/Core/BusinessObjectStringFormatterService.cs

```diff
@@ -116,9 +116,9 @@ public string GetPropertyString (IBusinessObject businessObject, IBusinessObject
       }
     }
 
-    private string GetStringValues (IBusinessObject businessObject, IBusinessObjectProperty property, string format, int lines)
+    private string GetStringValues (IBusinessObject businessObject, IBusinessObjectProperty property, string? format, int lines)
     {
-      IList list = (IList) businessObject.GetProperty (property);
+      IList? list = (IList?) businessObject.GetProperty (property);
```

Comment:
ah yes. here we handle null as a result of GetProperty. earlier, we have the null-bug, so we need to forgive this at the earlier spot, too. please, don't forget the tests for the new null fallbacks.